### PR TITLE
Remove deprecated `ADDRESS` environment variable

### PR DESCRIPTION
**Pull request by ysimonson**

### CODE REVIEW

**Comment by Tryneus** [comment]
Looks like you can also remove a small bit of code from the top of `etc/deploy/gen_pachd_tls.sh`, otherwise LGTM

**Comment by ysimonson** [comment]
Good catch; fixed

**Comment by ysimonson** [comment]
Strange, I seem to have pulled in some unrelated changes. I'm not sure how, digging in now.

File: etc/deploy/gen_pachd_tls.sh [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  echo "the cert that this script generates" >/dev/fd/2
+if [[ -n "${PACHD_ADDRESS}" ]]; then
+  echo "must run 'unset PACHD_ADDRESS' to use this script's cert. These" >/dev/fd/2
+  echo "variables prevent pachctl from trusting the cert that this script" >/dev/fd/2
+  echo "generates" >/dev/fd/2
   exit 1
 else
-  echo "Note that \$ADDRESS and \$PACHD_ADDRESS prevent pachctl from trusting "
-  echo "the cert that this script generates--do not set them"
+  echo "Note that \$PACHD_ADDRESS prevent pachctl from trusting the cert that"
```
Comment (Tryneus) [inline etc/deploy/gen_pachd_tls.sh]: nit: `prevent` -> `prevents`

**Comment by Tryneus** [comment]
I think your merge commit doesn't indicate it's from `master`, so github has the wrong base commit to diff against.

**Pull request merged**